lizmap/modules/lizmap/classes/qgisVectorLayerDatasource.class.php

```diff
@@ -33,6 +33,10 @@ class qgisVectorLayerDatasource
         'sql' => ' sql=(.*)$',
     );
 
+    protected $provider;
+
+    protected $datasource;
```

Comment:
**PhpStan** Say that ``` Access to an undefined property qgisVectorLayerDatasource::$provider. ``` and ```
 Access to an undefined property qgisVectorLayerDatasource::$datasource.``` [for this two line](https://github.com/3liz/lizmap-web-client/pull/2041/commits/76e291380881b866d6e079da0a8fbcbf0160c350#diff-520ea96b02586d47aa1d3ab69edf6f419050a52814f16d8f6e8e9a4f00d53bd7L51-L52)